src/components/Client/ClientOrder.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { orderTypes } from '../../utils/configs';
+import axios from 'axios';
+
+const ClientOrder = () => {
+    let history = useHistory();
+
+    const onSelect = (order) => {
+        axios.post('/client/order', {
+            order: order
+        })
+            .then(function (response) {
+                console.log(response);
+            })
+            .catch(function (error) {
+                console.log(error);
+            });
+        history.push('/taxi')
```

Comment:
always navigating to `/taxi` route? put this code inside `then` callback

Reply:
as I wanted to test the navigation functionality before the axois request part was ready, I temporarily put the history navigation outside of the then callback.